HW_5/main.py

```diff
@@ -0,0 +1,67 @@
+from fastapi import FastAPI, HTTPException, Request
+from fastapi.responses import HTMLResponse
+from pydantic import BaseModel
+from fastapi.templating import Jinja2Templates
+
+
+class Task(BaseModel):
+    id: int
+    title: str
+    description: str
+    status: str
+
+app = FastAPI()
+templates = Jinja2Templates(directory='HW_5/templates')
```

Comment:
У меня была ошибка
![image](https://github.com/Fejanin/HW_Flask_and_FastAPI/assets/49741194/179d574a-0eec-4f8a-83ec-dddb6f869dad)
Пока не исправил `templates`
```suggestion
templates = Jinja2Templates(directory='templates')
```
В остальном все верно

Reply:
Странно, у меня все работает (несколько раз перепроверил). Возможно разница в том откуда запускать проект?

Reply:
Возможно